src/SlashCommands.tsx

```diff
@@ -658,69 +657,6 @@ export const Commands = [
-        description: _td("slash_command|verify"),
-        runFn: function (cli, roomId, threadId, args) {
-            if (args) {
-                const matches = args.match(/^(\S+) +(\S+) +(\S+)$/);
-                if (matches) {
-                    const userId = matches[1];
-                    const deviceId = matches[2];
-                    const fingerprint = matches[3];
-
-                    return success(
-                        (async (): Promise<void> => {
-                            const device = await getDeviceCryptoInfo(cli, userId, deviceId);
-                            if (!device) {
-                                throw new UserFriendlyError("slash_command|verify_unknown_pair", {
-                                    userId,
-                                    deviceId,
-                                    cause: undefined,
-                                });
-                            }
-                            const deviceTrust = await cli.getCrypto()?.getDeviceVerificationStatus(userId, deviceId);
-
-                            if (deviceTrust?.isVerified()) {
-                                if (device.getFingerprint() === fingerprint) {
-                                    throw new UserFriendlyError("slash_command|verify_nop");
-                                } else {
-                                    throw new UserFriendlyError("slash_command|verify_nop_warning_mismatch");
-                                }
-                            }
-
-                            if (device.getFingerprint() !== fingerprint) {
-                                const fprint = device.getFingerprint();
-                                throw new UserFriendlyError("slash_command|verify_mismatch", {
-                                    fprint,
-                                    userId,
-                                    deviceId,
-                                    fingerprint,
-                                    cause: undefined,
-                                });
-                            }
-
-                            await cli.setDeviceVerified(userId, deviceId, true);
-
-                            // Tell the user we verified everything
-                            Modal.createDialog(InfoDialog, {
-                                title: _t("slash_command|verify_success_title"),
-                                description: (
-                                    <div>
-                                        <p>{_t("slash_command|verify_success_description", { userId, deviceId })}</p>
-                                    </div>
-                                ),
-                            });
-                        })(),
-                    );
-                }
-            }
-            return reject(this.getUsage());
-        },
-        category: CommandCategories.advanced,
-        renderingTypes: [TimelineRenderingType.Room],
-    }),
```

Comment:
Just setting a device as verified is not a thing supported by the rust-sdk? So `/verify` has just never worked with rust crypto? We might want to find a way to get this mentioned in the release notes either way.

Reply:
Hmm, I can change the PR title to show that.

Reply:
Perfect :)